pkg/integrations/oracledb_exporter/oracledb_exporter.go

```diff
@@ -0,0 +1,103 @@
+package oracledbexporter
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	oe "github.com/observiq/oracledb_exporter/collector"
+
+	// required driver for integration
+	_ "github.com/sijms/go-ora/v2"
+	"github.com/xo/dburl"
+)
+
+// DefaultConfig is the default config for the oracledb v2 integration
+var DefaultConfig = Config{
+	ConnectionString: os.Getenv("DATA_SOURCE_NAME"),
+	MaxOpenConns:     10,
+	MaxIdleConns:     0,
+	QueryTimeout:     5,
+	ScrapeInterval:   0,
+}
+
+var (
+	errNoConnectionString = errors.New("no connection string was provided")
+)
+
+// Config is the configuration for the oracledb v2 integration
+type Config struct {
+	ConnectionString  string        `yaml:"connection_string"`
+	MaxIdleConns      int           `yaml:"max_idle_connections"`
+	MaxOpenConns      int           `yaml:"max_open_connections"`
+	ScrapeInterval    time.Duration `yaml:"scrape_interval"`
```

Comment:
This collides with the common config's `scrape_interval`, I think this should be under a separate key (the common config's `scrape_interval` is about autoscrape, this refers to time between refresh of metrics, if I understand correctly.)